pkgs/tools/package-management/rpm/default.nix

```diff
@@ -6,11 +6,11 @@
 
 stdenv.mkDerivation rec {
   pname = "rpm";
-  version = "4.17.0";
+  version = "4.17.1";
 
   src = fetchurl {
-    url = "http://ftp.rpm.org/releases/rpm-${lib.versions.majorMinor version}.x/rpm-${version}.tar.bz2";
-    sha256 = "2e0d220b24749b17810ed181ac1ed005a56bbb6bc8ac429c21f314068dc65e6a";
+    url = "https://ftp.osuosl.org/pub/rpm/releases/rpm-${lib.versions.majorMinor version}.x/rpm-${version}.tar.bz2";
```

Comment:
Why aren't we using official rpm site ?

Reply:
Probably because it didn't have the version at the time. I don't really remember. Ideally we could use a `mirrors://` proto.